test/fixtures/os/sles_15/etc/fs/os-release

```diff
@@ -0,0 +1,8 @@
+NAME="SLES"
+VERSION="15"
+VERSION_ID="15"
+PRETTY_NAME="SUSE Linux Enterprise Server 15"
+ID="sles"
+ID_LIKE="suse"
+ANSI_COLOR="0;32"
+CPE_NAME="cpe:/o:suse:sles:15"
```

Comment:
Should we include an empty line in the end?

Reply:
I moved the file and added newline